src/workflows-app/components/FilterableWorkflowTable.ts

```diff
@@ -151,6 +206,57 @@ const FilterableWorkflowTable = ({
     }
   };
 
+  const includeKeys = useMemo(
+    () => [
+      'backendStatus',
+      'executionStatus',
+      'shardIndex',
+      'jobId',
+      'start',
+      'end',
+      'stderr',
+      'stdout',
+      'tes_stdout',
+      'tes_stderr',
+      'attempt',
+      'subWorkflowId', // needed for task type column
+    ],
+    []
+  );
+  const excludeKeys = useMemo(() => [], []);
+
+  const stripTaskPrefixFromKeys = (keyValuePairs: {}): {} => {
+    return Object.fromEntries(
+      Object.entries(keyValuePairs).map(([key, value]) => [parseMethodString(key).variable, value])
+    );
+  };
+
+  const loadWorkflows = useCallback(
+    async (workflowId: string | undefined): Promise<WorkflowMetadata | undefined> => {
+      if (workflowId === undefined) {
+        return undefined;
+      }
+
+      try {
+        const { cromwellProxyUrlState } = await loadAppUrls(workspaceId, 'cromwellProxyUrlState');
+        if (cromwellProxyUrlState.status === AppProxyUrlStatus.Ready) {
+          return await fetchMetadata({
+            cromwellProxyUrl: cromwellProxyUrlState.state,
```

Comment:
Is there a way to cache this so we don't have to make the same request again each time the user clicks a different link on the same workflow?